CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+{% for section, _ in sections.items() %}
```

Comment:
I haven't worked with towncrier, but why do we have jinja templates in CHANGELOG.md, which will be auto-updated by towncrier?

Reply:
I have no idea how template code ended up in this file. It's not in issue by itself (because it'll be overwritten anyway), but the file shouldn't look like this. Fixed.